Sources/StreamChat/Workers/MessageUpdater.swift

```diff
@@ -192,6 +192,7 @@ class MessageUpdater: Worker {
                 extraData: extraData
             )
 
+            newMessageDTO.showInsideThread = true
```

Comment:
Does this always have to be true? Does it impact the case when "send in channel" is true?

Reply:
No this only impacts when showing replies inside the thread 👍